### ISSUE TITLE
Crash on npx prisma migrate save --experimental

### ISSUE THREAD

**Issue by chadwhittaker** [prisma/migrate#658]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v14.15.0           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [libs/sql-schema-describer/src/walkers.rs:214:27] index out of bounds: the len is 0 but the index is 0

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

**Comment by ganobrega** [comment]
Hello @chadwhittaker!
What provider you are using?

**Comment by tomhoule** [comment]
I think this should have been fixed in 2.12.0. Does it still happen with a newer version of the CLI?

**Comment by tomhoule** [comment]
Closing for inactivity. Please open a new issue if the problem manifests in a new migration. Thanks!
